spec/classes/profile/known_host_public_keys_spec.rb

```diff
@@ -8,13 +8,15 @@
 describe "nebula::profile::known_host_public_keys" do
   on_supported_os(supported_os: Nebula.supported_os).each do |os, os_facts|
     context "on #{os}" do
+      let(:my_ip) { Faker::Internet.ip_v4_address }
```

Comment:
Is `:my_ip` a pattern from something? Just in this PR, I'm only seeing it being assigned to `networking.ip`, so I'm left wondering what it's for. Like, if I saw us checking against `my_ip` later in the tests, that would make sense, but if not, why not just set `networking.ip` to something invalid like `0.1.2.3`?

Not requesting a change or anything, just wondering if I'm missing anything.